javascript-forms/form-values.js

```diff
@@ -0,0 +1,11 @@
+$contactForm = document.forms[0]
+$contactForm.addEventListener("submit", function(event){
+  event.preventDefault();
+  var formValues = {
+    $name: $contactForm.elements.name.value,
```

Comment:
These properties are not holding elements, so a `$` in the name would not be necessary for these 👍